### PR TITLE
LPS 125499

### PR DESCRIPTION
**Pull request by carolmariaabb**
https://issues.liferay.com/browse/LPS-125499

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:forms

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DocumentLibrary/DocumentLibrary.es.js [hunk 1]
```diff
@@ -399,22 +403,11 @@ const Main = ({
...
-					setCurrentValue(JSON.stringify(file));
-
-					onChange(event, JSON.stringify(file));
-				}
-
+				handleGuestUploadFileChanged(
+					error ? error.message : '',
+					event,
+					error ? null : JSON.stringify(file)
+				);
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DocumentLibrary/DocumentLibrary.es.js:410]: @carolmariaabb, let's avoid use ternary operators on the methods parameters in order to keep a good code readability. 😉 

One more thing, please keep the methods invocation separated by line breaks.

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Comment by carolmariaabb** [comment]
Resent [here](https://github.com/natocesarrego/liferay-portal/pull/963).

**Pull request closed without merging**